websites/config_schema/site-config-schema.yml

```diff
@@ -31,6 +31,7 @@ field:
     condition: include('field_condition', required=False)
     display_field: str(required=False)
     collection: str(required=False)
+    collections: list(str(), required=False)
```

Comment:
What is this for?

Reply:
@gsidebo is this required for the PR? I'm confused that there's a `collection` and `collections`

Reply:
I initially implemented this in a way that required the site config to point directly to the types that would be used for internal links. For example, the config would have to include `"collections": ["page", "resource"]` to indicate that only those items should be included in the list of available internal links. Instead of that we're just allowing all page content in that list, but the logic was implemented and could be useful in the future. You can test it by adding `"collections": ["page"]` et al to a menu field in the site config